Documentation/AuthConflictsBetweenSDKs.md

```diff
@@ -0,0 +1,42 @@
+# Bridging gap between old and new SDKs
+
+Authentication conflicts can happen due to mixing new and old azure SDKs. The credential types created by the new authentication library (Microsoft.Rest.ClientRuntime.Azure.Authentication, type = ServiceClientCredentials)  are incompatible with the credential types  used in older SDKs (Microsoft.Azure.Common, type = SubscriptionCloudCredentials). The problem is that using two different authentication libraries will require you to authenticate twice, which is a painful experience.   
```

Comment:
Need to flesh out what is meant by new and old SDKs.  You can list the old SDKs, and specify that SDKs with depend on Microsoft.Azure.Common are old and SDKs which depend on Microsoft.Rest.ClientRuntime are new